src/main/kotlin/org/jitsi/nlj/transform/node/ExclusivePathDemuxer.kt

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright @ 2018 - present 8x8, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jitsi.nlj.transform.node
+
+import org.jitsi.nlj.PacketInfo
+
+/**
+ * Packets are passed only to the first path which accepts them
+ */
+class ExclusivePathDemuxer(name: String) : DemuxerNode(name) {
+    override fun doProcessPackets(p: List<PacketInfo>) {
+        p.forEach packets@ { packetInfo ->
+            transformPaths.forEach { conditionalPath ->
```

Comment:
Is this going to preserve the order of the paths added to the demuxer? Would it be worth it to put the video path before the audio path in the audio/video demuxer, since video will be hit more often?